website/docs/r/kvstore_instance.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_kvstore_instance"
+sidebar_current: "docs-alicloud-resource-kvstore-instance"
+description: |-
+  Provides an ApsaraDB Redis / Memcache instance resource.
+---
+
+# alicloud\_kvstore\_instance
+
+Provides an ApsaraDB Redis / Memcache instance resource. A DB instance is an isolated database environment in the cloud. It can be associated with IP whitelists and backup configuration which are separate resource providers.
+
+## Example Usage
+
+```
+resource "alicloud_kvstore_instance" "default" {
+  instance_class = "redis.master.small.default"
+  instance_name  = "myredis"
+  password       = "Passw0rd"
+  vswitch_id     = "some vswitch id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `instance_name` - (Optional) The name of DB instance. It a string of 2 to 256 characters.
+* `password`- (Required) The password of the DB instance. The password is a string of 8 to 30 characters and must contain uppercase letters, lowercase letters, and numbers. 
+* `instance_class` - (Required) Type of the applied ApsaraDB for Redis instance.
+For more information, see [Instance type table](https://www.alibabacloud.com/help/doc-detail/61135.htm?spm=a2c63.p38356.a3.3.429a59abAfUku0).
```

Comment:
please remove the suffix '?spm=a2c63.p38356.a3.3.429a59abAfUku0'

Reply:
done